### ISSUE TITLE
Missing the `aria-label` attribute from the `BitSticky` component

### ISSUE THREAD

**Issue by Cyrus-Sushiant** [bitfoundation/bitplatform#8041]
To improve accessibility, we need to add the `aria-label` attribute to the `BitSticky` component. This will ensure that screen readers can provide a meaningful description of the component for users with disabilities.
